content/en/security/guide/tls_deprecation_1_2.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Deprecation notice for TLS version < 1.2
+kind: guide
+---
+
+
+## Overview
+
+Transport Layer Security (TLS) is a critical security protocol used to protect web traffic. It provides confidentiality and integrity of data in transit between clients and servers exchanging information. Datadog is disabling support for older versions of TLS, below 1.2 (SSLv3, TLS v1.0, TLS v1.2) across public facing Datadog applications, beginning March 31, 2022. If you use unsupported clients to connect to Datadog after the older protocols are disabled, you will receive connection error messages.
+
+### Reason for deprecation
+
+These protocols are being deprecated to ensure that customers connect to Datadog using secure connection channels. This is in accordance with a decision from the Internet Engineering Task Force (IETF) to deprecate these protocols as of March 25, 2021. ([https://datatracker.ietf.org/doc/rfc8996/][1])
+
+## Browser support
+
+Modern browsers have had support for TLS 1.2 for a while. See the "Can I use..." [compatibility matrix][2] to determine if your specific browser and version are affected.
+
```

Comment:
Adding a section about Agent support:

```suggestion
## Agent support

### Agent v6 and v7

All versions of Agent v6 and v7 support TLS 1.2.

### Agent v5

#### Packaged/containerized Agent v5

All versions of the Agent v5 installed with:

* the DEB/RPM packages
* the Windows MSI installer
* the official container image

support TLS 1.2.

#### Agent v5 source install

When installed with the [source install script](https://github.com/DataDog/dd-agent/blob/5.32.8/packaging/datadog-agent/source/setup_agent.sh), the Agent v5 relies on the system's Python and OpenSSL. Therefore, support for TLS 1.2 depends on the versions of Python and OpenSSL installed on the system.

To determine if your system's Python supports TLS 1.2 (and therefore if the source-installed Agent supports TLS 1.2), run: `python -c "import json, urllib2; print json.load(urllib2.urlopen('https://www.howsmyssl.com/a/check'))['tls_version']"` from a system shell, which outputs `TLS 1.2` if TLS 1.2 is supported, and an older TLS version or an error otherwise. If TLS 1.2 is not supported, upgrade your system's Python and OpenSSL or upgrade the Agent to v7.
```

Reply:
See my suggestion for Python below. Let's add change that to:

> To determine if your system's Python supports TLS 1.2 (and therefore if the source-installed Agent supports TLS 1.2) see the Python section below. If TLS 1.2 is not supported, upgrade your system's Python and OpenSSL or upgrade the Agent to v7.